packages/forms/src/Components/Builder.php

```diff
@@ -831,6 +831,23 @@ public function isBlockLabelTruncated(): bool
         return (bool) $this->evaluate($this->isBlockLabelTruncated);
     }
 
+    /**
+     * @return Component[]
```

Comment:
```suggestion
     * @return array<Block>
```

Reply:
@danharrin The method is returning `Component` instead of Block:

```  841    Method Filament\Forms\Components\Builder::getBlockPickerBlocks() should return array<Filament\Forms\Components\Builder\Block> but returns
         array<Filament\Forms\Components\Component>
```

So I used `@return array<Component>` instead

Reply:
It actually does return blocks though, there are ways we can convince PHPstan of that with a `@var` block